addons/mail/static/src/models/composer/composer.js

```diff
@@ -567,6 +565,9 @@ function factory(dependencies) {
         isDoFocus: attr({
             default: false,
         }),
+        isLog: attr({
+            default: false,
+        }),
```

Comment:
It was already the case before but I feel like this is not a "real boolean" and is has not a meaningful name.

If there was a 3rd state (other than note and comment) this wouldn't be a boolean.

I would be in favor of renaming this to "messageSubtype" and have "note" or "comment" as possible values. Maybe even the actual `subtype_xmlid` though maybe that is going too far/coupling both concepts too much.

Reply:
Seems logical to me but before taking the time to change that, I prefer to be sure that @alexkuhn agree with this ?
(so I'm not reverting the change later on :stuck_out_tongue: )

Reply:
I wouldn't make this change, because this is confusing having `messageSubtype` as an `attr` on `composer`. It's best keeping configuration fields as simple as possible, and here it's good enough.

I would be more confident to make such a change with an improved modeling, like considering "content" of composer being a message "in progress" and kinda in an "editable". That way, this would make a lot of sense to merge both concepts `isLog` and `subtype`.

Reply:
Ok, should just create a new task for that last idea then.

Reply:
task-2283218